internal/bsr/ssh/types.go

```diff
@@ -69,16 +70,25 @@ const (
 	FileTransferDownload      FileTransferDirection = "download"
 )
 
+// OpenChannelError provides details if a channel was rejected.
+// This will contain details from the SSH_MSG_CHANNEL_OPEN_FAILURE request
+// that rejected the channel.
+// See: https://www.rfc-editor.org/rfc/rfc4254#section-5.1
+type OpenChannelError ssh.OpenChannelError
```

Comment:
I defined the type to make it easier to expand if we want to capture additional information in the future. For example we might want a string version of the reason code instead of just the uint32. It also provides a good spot to document what this is.